small-talk/app/accountCreate/page.js

```diff
@@ -1,6 +1,8 @@
 "use client"
 import Link from 'next/link'
 import React, { useState } from 'react';
+import handleCreate from "@/db/hcTest"
```

Comment:
Also had another thought. Maybe we should rename from Test since they're not being used in the testing suite

Reply:
Just completed that idea. Thank you for that